pkgs/applications/kde/signon-kwallet-extension.nix

```diff
@@ -0,0 +1,45 @@
+{
+  stdenv,
+  mkDerivation,
+  extra-cmake-modules,
+  symlinkJoin,
+  kwallet,
+  signond
+}:
+
+mkDerivation rec {
+
+  name = "signon-kwallet-extension";
+
+  preConfigure = ''
+    # mock signond plugins dir
+    sed 's/set(SIGNONEXTENSION_PLUGINDIR ''${_pkgconfig_invoke_result})/set(SIGNONEXTENSION_PLUGINDIR lib\/signon\/extensions )/g' -i cmake/modules/FindSignOnExtension.cmake
```

Comment:
If you're replacing text verbatim (not using a regular expression) it's preferred to use [substituteInPlace](https://nixos.org/nixpkgs/manual/#ssec-stdenv-functions) because `sed` does not fail if there are no matches.

Reply:
Fixed